pyteal/ast/router.py

```diff
@@ -203,6 +204,10 @@ class BareCallActions:
         kw_only=True, default=OnCompleteAction.never()
     )
 
+    def __post_init__(self):
```

Comment:
bring up the validation so can tell that bare calls are messed up before starting to build AST